daemon/docker_watcher.go

```diff
@@ -344,45 +338,11 @@ func (d *Daemon) retrieveDockerLabels(dockerID string) (*dTypes.ContainerJSON, l
 	return &dockerCont, newLabels, nil
 }
 
-func updateOrchLabels(c *container.Container, l labels.Labels) bool {
-	changed := false
-
-	c.OpLabels.Orchestration.MarkAllForDeletion()
-	c.OpLabels.Disabled.MarkAllForDeletion()
-
-	for k, v := range l {
-		if c.OpLabels.Disabled[k] != nil {
-			c.OpLabels.Disabled[k].DeletionMark = false
-		} else {
-			if c.OpLabels.Orchestration[k] != nil {
-				c.OpLabels.Orchestration[k].DeletionMark = false
-			} else {
-				tmp := v.DeepCopy()
-				log.Debugf("Assigning orchestration label %+v", tmp)
-				c.OpLabels.Orchestration[k] = tmp
-				changed = true
-			}
-		}
-	}
-
-	if c.OpLabels.Orchestration.DeleteMarked() || c.OpLabels.Disabled.DeleteMarked() {
-		changed = true
-	}
-
-	return changed
-}
-
 func (d *Daemon) deleteContainer(dockerID string) {
 	log.Debugf("Processing deletion event for docker container %s", dockerID)
 
 	d.containersMU.Lock()
-	if container, ok := d.containers[dockerID]; ok {
-		sha256sum := container.OpLabels.Enabled().SHA256Sum()
-		if err := d.DeleteIdentityBySHA256(sha256sum, dockerID); err != nil {
-			log.Errorf("Error while deleting labels (SHA256SUM:%s) %+v: %s",
-				sha256sum, container.OpLabels.Enabled(), err)
-		}
-
+	if _, ok := d.containers[dockerID]; ok {
```

Comment:
Redundant, you can remove this line as well and simply write `delete(d.containers. dockerID)`